src/components/entry/entry.scss

```diff
@@ -1,3 +1,12 @@
+button {
+  all: unset;
+  cursor: pointer;
```

Comment:
is not the default for a button?

Reply:
I used unset to clear all the default styles because i don't want it to look like a button